src/omotes_sdk/internal/common/broker_interface.py

```diff
@@ -291,9 +291,9 @@ async def _stop_broker_interface(self) -> None:
         )
         for queue_task in tasks_to_cancel:
             queue_task.cancel()
-        if self._channel:
+        if hasattr(self, "_channel") and self._channel:
             await self._channel.close()
-        if self._connection:
+        if hasattr(self, "_connection") and self._connection:
```

Comment:
Added as in limited occassions it can happen that the setup of the connection files and this field is not yet set.